Competition2017/tests/competition/subsystems/feeder/TestFeederSubsystem.java

```diff
@@ -0,0 +1,33 @@
+package competition.subsystems.feeder;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.feeder.SideFeederSubsystem.FeederSide;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.XPropertyManager;
+
+public class TestFeederSubsystem extends BaseSubsystem {
+
+    private TestSideFeederSubsystem leftFeeder;
+    private TestSideFeederSubsystem rightFeeder;
+
+    @Inject
+    public TestFeederSubsystem(WPIFactory factory, XPropertyManager propManager){
+
+        leftFeeder = new TestSideFeederSubsystem(31, FeederSide.Left, factory, propManager);
```

Comment:
Can you explain further? I don't see why we can't do here what was done with the `SideFeederSubsystem`: make it inherit from the real version and call `super`.